man/man5/zfs-module-parameters.5

```diff
@@ -1161,6 +1172,71 @@ Rate limit delay zevents (which report slow I/Os) to this many per second.
 Default value: 20
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_unflushed_max_mem_amt\fR (ulong)
+.ad
+.RS 12n
+Upper-bound limit for unflushed metadata changes to be held by the
+log spacemap in memory (in bytes).
+.sp
+Default value: \fB1,073,741,824\fR (1GB).
+.RE
+
+.sp
+.ne 2
+.na
+\fBzfs_unflushed_max_mem_ppm\fR (ulong)
+.ad
+.RS 12n
+Percentage of the overall system memory that ZFS allows to be used
+for unflushed metadata changes by the log spacemap.
+(value is calculated over 1000000 for finer granularity).
+.sp
+Default value: \fB1000\fR (which is divided by 1000000, resulting in
+the limit to be \fB0.1\fR% of memory)
+.RE
+
+.sp
+.ne 2
+.na
+\fBzfs_unflushed_log_block_max\fR (ulong)
+.ad
+.RS 12n
+Upper-bound that limits the size of the log spacemap, described in
+terms of 128K blocks.
+.sp
+Default value: \fB262144\fR (256K).
```

Comment:
I'm not sure this description is right.  The code seems to treat this as a block count (of 128K blocks), so the default value of 262144 means that the space in all the log spacemaps can add up to no more than 262144 blocks (or 32GB of logical space (meaning before compression and ditto blocks)).  This might be a good place to write about the considerations around changing this.  E.g. the impact on unclean pool import time vs flush frequency.  Wouldn't hurt to explain why it's in blocks rather than bytes, too.